handlers/ci.go

```diff
@@ -1,29 +1,179 @@
 package handlers
 
 import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strings"
 
 	"github.com/golang/glog"
 )
 
-type CircleCIInfo struct {
-	CircleJob string `json:"build_parameters[CIRCLE_JOB]"`
-	Revision  string `json:"revision"`
+const (
+	TravisCIEndPoint = "https://api.travis-ci.org" //travis-ci endpoint
+	ContentTypeJSON  = "application/json"          //content-type
+	TravisAPIVersion = "Travis-API-Version"        //API version, a Mandatory header field for Travis-CI V3 APIs
+	Build            = "build"                     //Job name
```

Comment:
these are job types . better we move them to a separate group. something like this : 

```go
type TravisJobType string

const (
    Verify TravisJobType = "verify"
    UnitTest = "unittest"
    .....
)
```